clients/python-pyo3/README.md

```diff
@@ -12,144 +12,180 @@
 **[API Reference](https://www.tensorzero.com/docs/gateway/api-reference)** ·
 **[Configuration Reference](https://www.tensorzero.com/docs/gateway/deployment)**
 
-This is the future version of the Python client, which includes the Rust client as a native library
-
-The `tensorzero` package provides an async Python client for the TensorZero Gateway.
+The `tensorzero` package provides a Python client for the TensorZero Gateway.
 This client allows you to easily make inference requests and assign feedback to them via the gateway.
 
 See our **[API Reference](https://www.tensorzero.com/docs/gateway/api-reference)** for more information.
 
 ## Installation
 
-This client is currently not published on `PyPI` - for production code, you most likely want to
-use the Python client in `<repository root>/clients/python`
-
-To try out the new Rust-based client implementation, run:
-
-```bash
-pip install -r requirements.txt
-maturin develop
-```
-
-If using `uv`, then instead run:
-
 ```bash
-uv venv
-uv pip sync requirements.txt
-uv run maturin develop --uv
-uv run python
+pip install tensorzero
 ```
 
 ## Basic Usage
 
-### Initializing the client
+### Initialization
 
-The TensorZero client comes in both synchronous (`TensorZeroGateway`) and asynchronous (`AsyncTensorZeroGateway`) variants.
-Each of these classes can be constructed in one of two ways:
+The TensorZero client offers synchronous (`TensorZeroGateway`) and asynchronous (`AsyncTensorZeroGateway`) variants.
+Additionally, the client can launch an embedded (in-memory) gateway (`build_embedded`) or connect to an external HTTP gateway (`build_http`).
```

Comment:
I can't find it but maybe it is lost in the diff: we should be sure to explicitly say that the methods are the same on each variant of the client